doc/man3/EVP_DigestSignInit.pod

```diff
@@ -31,6 +31,10 @@ B<*pctx> is overwritten. The EVP_PKEY_CTX value returned must not be freed
 directly by the application (it will be freed automatically when the EVP_MD_CTX
 is freed). The digest B<type> may be NULL if the signing algorithm supports it.
 
+There is no B<EVP_PKEY_CTX> will be created by EVP_DigestSignInit() if the passed
+B<ctx> is already assigned with an B<EVP_PKEY_CTX>, take L<EVP_MD_CTX_set_pkey_ctx(3)>
+and L<SM2(7)> as references.
```

Comment:
Oh...but wait...there is no SM2(7)!? There probably needs to be something (maybe not in this PR) - otherwise how will people know about the special hoops they need to jump through to be able to use SM2?

Reply:
In fact I forgot to include that SM2(7)... I used 'git add -u' so the new SM2.pod file is left on my disk...now added